### PR TITLE
[ACR] Fix some bugs on uncovered cmds

### PR DESCRIPTION
**Pull request by rosanch**
**Description**
I was still able to find some bugs related to the SDK update.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
ACR

File: src/azure-cli/azure/cli/command_modules/acr/scope_map.py [hunk 1]
```diff
@@ -39,10 +39,12 @@ def acr_scope_map_create(cmd,
...
     actions = parse_scope_map_actions(repository_actions_list, gateway_actions_list)
 
-    scope_map = {
-        'actions': actions,
-        'description': description
-    }
+    ScopeMap = cmd.get_models('ScopeMap')
+
+    scope_map = ScopeMap(
+        actions = actions,
```
Comment (00Kai0) [inline src/azure-cli/azure/cli/command_modules/acr/scope_map.py:45]: remove unexpected spaces:
```
    scope_map = ScopeMap(
        actions=actions,
        description=description
    )
```

**Review by 00Kai0 — CHANGES_REQUESTED** [review]

**Comment by 00Kai0** [comment]
LGTM

File: src/azure-cli/azure/cli/command_modules/acr/scope_map.py [hunk 2]
```diff
@@ -39,10 +39,12 @@ def acr_scope_map_create(cmd,
...
+        description = description
```
Comment (jsntcy) [inline src/azure-cli/azure/cli/command_modules/acr/scope_map.py:46]:
```suggestion
        actions=actions,
        description=description
```

**Comment by zhoxing-ms** [comment]
There are no any recorded yaml files here. May I ask have you rerun the relevant tests?

**Comment by zhoxing-ms** [comment]
By the way, the reason why these bugs have not been found before is because the tests do not cover these cases? If so, please add relevant tests~

**Review by zhoxing-ms — CHANGES_REQUESTED** [review]
I'd like to ask is this PR for hotfix? If so, please submit hotfix related PRs to the release branch. Please refer to this document for details [doc link](https://github.com/Azure/azure-cli/blob/main/doc/authoring_command_modules/README.md#hotfix-pr)

**Review by 00Kai0 — APPROVED** [review]

**Review by northtyphoon — APPROVED** [review]
LGTM

**Comment by rosanch** [comment]
Connected-registry currently has no test at all.
Scope Map I'm not sure why it didn't have an update command covered in the tests. It seems it was token was the main group covered in the testing so scope-map wasn't fully covered.

**Comment by rosanch** [comment]
@zhoxing-ms we want to include this in the hot release. But on top of the PR #18786 and I don't see that merged to the release branch. Wouldn't it make some merge conflict if I merge this change first?
cc: @northtyphoon

**Comment by zhoxing-ms** [comment]
This change has been merged into the release branch by https://github.com/Azure/azure-cli/pull/18853. After hotfix is released, we will merge back to the dev branch, so close this PR first

**Comment by 00Kai0** [comment]
Close it, because PR https://github.com/Azure/azure-cli/pull/18853 has included it.

**Pull request closed without merging**